### ISSUE TITLE
`compat` admonition blocks need to be rendered somehow

### ISSUE THREAD

**Issue by asinghvi17** [LuxDL/DocumenterVitepress.jl#95]
Apparently this is a thing...
```
!!! compat Julia 1.11

	Scoped values were introduced in Julia 1.11. In Julia 1.8+ a compatible implementation is available from the package 		 
    ScopedValues.jl.

```

**Comment by lazarusA** [comment]
not sure this is resolve. In principle we could defined a custom admonition, i.e. `::: compat ....`. And the issue is that if this given name is not in default list, then it would default back to `tip`, which is not ideal.

**Comment by lazarusA** [comment]
It looks like the compat admonition is different from `tip`.  But, for now this is a very good way to move forward. And yes, the creating will need to happen at the vitepress level. 

https://documenter.juliadocs.org/stable/showcase/#Compat-admonition

**Comment by asinghvi17** [comment]
Good point!  I guess we could add some CSS for that?  The current styling seems to be:
```
<div class="tip custom-block">
```
so somehow making "compat custom-block" inherit from "tip custom-block" in CSS and changing the colors ought to work.
